.github/workflows/ci.yml

```diff
@@ -37,21 +37,17 @@ jobs:
     - name: Fix git protocol for Node 14
       if: ${{ startsWith(matrix.NODE_VERSION, '14.') }}
       run: sudo git config --system url."https://github".insteadOf "ssh://git@github"
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node.js
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.NODE_VERSION }}
-    - name: Cache Node.js modules
-      uses: actions/cache@v2
+        cache: 'npm'
+    - name: Start MongoDB
+      uses: supercharge/mongodb-github-action@1.8.0
       with:
-        path: ~/.npm
```

Comment:
See https://github.com/parse-community/parse-server/issues/7614#issuecomment-945178352

Reply:
I don't think it really matters here. Technically `actions/cache` used here creates 1 cache for all node versions. Look at the key and restore key.